docs/GOVERNANCE.md

```diff
@@ -99,16 +99,18 @@ If a vote is taken during a meeting, the follow rules will be followed:
   An acceptable absence would include situations where the person is not
   officially working at all, such as being on vacation, taking a sabbatical or
   there is a public holiday. However, situations such as a scheduling conflict
-  would not apply. Absence from meetings during that time will not impact
-  their voting rights.
-- Only members with voting rights will be allowed to vote.
+  would not apply. Absence from meetings during a leave-of-absence will not
+  impact their voting rights.
+- Only members with voting rights will have thier vote counted towards the
```

Comment:
Later on we say other folks can vote, but we should be clear that their vote doesn't count. Just voting members' votes count.

Reply:
fix "thier"